### ISSUE TITLE
OPTIONS request bodies (Trac #98)

### ISSUE THREAD

**Issue by mnot** [httpwg/httpbis-issues#98]
This text in 9.2:

A server that does not support such an extension MAY discard the request body.

needs to be aligned with the resolution of #19; e.g.,

An origin server that does not...

Migrated from https://trac.ietf.org/ticket/98
```json
{
    "status": "closed",
    "changetime": "2008-07-14T12:18:11",
    "_ts": "1216037891000000",
    "description": "This text in 9.2:\n\nA server that does not support such an extension MAY discard the request body.\n\nneeds to be aligned with the resolution of #19; e.g.,\n\nAn origin server that does not...\n\n\n",
    "reporter": "mnot@pobox.com",
    "cc": "",
    "resolution": "fixed",
    "time": "2008-02-05T07:00:58",
    "component": "p2-semantics",
    "summary": "OPTIONS request bodies",
    "priority": "",
    "keywords": "",
    "milestone": "04",
    "owner": "julian.reschke@gmx.de",
    "type": "editorial",
    "severity": ""
}
```

**Comment by mnot** [comment]
Trac update at 20080205T22:59:23: fielding@gbiv.com commented:

> The sentence is nonsense -- we can just delete it.
>

**Comment by mnot** [comment]
Trac update at 20080228T05:28:42: mnot@pobox.com changed milestone from "02" to "03"

**Comment by mnot** [comment]
Trac update at 20080530T15:02:00: julian.reschke@gmx.de changed owner from "" to "julian.reschke@gmx.de"

**Comment by mnot** [comment]
Trac update at 20080530T15:02:27: 
* julian.reschke@gmx.de changed attachment from "" to "i98.diff"
* julian.reschke@gmx.de commented:

> proposed change for part 2.

**Comment by mnot** [comment]
Trac update at 20080714T12:13:26: julian.reschke@gmx.de changed milestone from "03" to "04"

**Comment by mnot** [comment]
Trac update at 20080714T12:18:11: 
* julian.reschke@gmx.de commented:

> Fixed in [280]:
> 
> Resolve #98: remove superfluous statement about request body handling for OPTIONS (closes #98).


* julian.reschke@gmx.de changed resolution from "" to "fixed"
* julian.reschke@gmx.de changed status from "new" to "closed"
